h5p-bildetema/src/components/Toggle/Toggle.tsx

```diff
@@ -5,18 +5,20 @@ type ToggleProps = {
   handleChange: (checked: boolean) => void;
   checked: boolean;
   label?: string;
```

Comment:
Not strictly related to this PR, but we should _always_ have a label. It doesn't necessarily have to be visible, but visibly impaired users might need it.